src/java/picard/analysis/CollectRnaSeqMetrics.java

```diff
@@ -49,12 +49,54 @@
 import java.util.Set;
 
 @CommandLineProgramProperties(
-        usage = "Collect metrics about the alignment of RNA to various functional classes of loci in the genome:" +
-                "coding, intronic, UTR, intergenic, ribosomal. Also determines strand-specificity for strand-specific libraries.",
-        usageShort = "Produces RNA alignment metrics for a SAM or BAM file",
+        usage = CollectRnaSeqMetrics.USAGE_SUMMARY + CollectRnaSeqMetrics.USAGE_DETAILS,
+        usageShort = CollectRnaSeqMetrics.USAGE_SUMMARY,
         programGroup = Metrics.class
 )
 public class CollectRnaSeqMetrics extends SinglePassSamProgram {
+
+    static final String USAGE_SUMMARY = "Produces RNA alignment metrics for a SAM or BAM file.  ";
+    static final String USAGE_DETAILS = "Program takes aligned RNA transcript sequences and determines the functional distribution of bases " +
+            "within the transcripts of a BAM file including the numbers and percentages of bases that are located within " +
+            "the untranslated region (UTR), introns, intergenic (sequences between " +
+            "discrete genes), as well peptide-coding sequences.  This tool also calculates quality metrics on the reads and the bases within" +
+            " the reads including the numbers of bases that pass the sequencing platform quality filters (PF_BASES).  " +
+            "For additional information on the PF metric, please see the GATK dictionary entry on (PF Reads/Illumina Chastity Filters) " +
+            "at:" +
+            "<ul> http://www.broadinstitute.org/gatk/guide/topic?name=dictionary </ul>" +
```

Comment:
it was already noted that the use of html tags in the usage string leads to ugly commandline information...do we have a good solution for this?